bill/invoice.go

```diff
@@ -68,13 +62,13 @@ type Invoice struct {
 	// Expenses paid for by the supplier but invoiced directly to the customer.
 	Outlays Outlays `json:"outlays,omitempty" jsonschema:"title=Outlays"`
 
-	// Summary of all the invoice totals, including taxes.
-	Totals *Totals `json:"totals" jsonschema:"title=Totals"`
-
 	Ordering *Ordering `json:"ordering,omitempty" jsonschema:"title=Ordering Details"`
 	Payment  *Payment  `json:"payment,omitempty" jsonschema:"title=Payment Details"`
```

Comment:
Curiosity: Also, no possibility for multiple payment methods?  (part in cash, part credit, for example)

Reply:
Kind of... it's possible to define a list of acceptable methods, but only possible to define a single preferred method.

You have `pay.Instructions` for that: https://github.com/invopop/gobl/blob/main/pay/instructions.go#L42